### ISSUE TITLE
fix: RadioButtonGroup and RadioGroup keyboard navigation does not work on firefox and safari

### ISSUE THREAD

**Issue by TahimiLeonBravo** [nexxtway/react-rainbow#1156]
`it is a problem with focus, they do not get focus when clicking and then the keyboard does not work`

**Comment by TahimiLeonBravo** [comment]
@yvmunayev go inside this issue

**Comment by jonathanurias96** [comment]
Hello! May I work in this issue?

**Comment by TahimiLeonBravo** [comment]
> Hello! Can I work in this issue?

Great!! Go for it!

**Comment by jonathanurias96** [comment]
@TahimiLeonBravo  Hello! I noticed this is the styled-components code that fills the radio button when is clicked (or checked):
<img width="534" alt="Screen Shot 2020-06-29 at 1 40 53 PM" src="https://user-images.githubusercontent.com/57416786/86054097-62e98100-ba0e-11ea-962f-718ef77726d0.png">

I would like to add the same css attributes to the same radio button but instead of the checked attribute for reference I would like to add it when the tag has an attribute "aria-checked="true"". 
In regular css its done this way:
<img width="328" alt="Screen Shot 2020-06-29 at 1 46 42 PM" src="https://user-images.githubusercontent.com/57416786/86054499-20747400-ba0f-11ea-85c5-cbf0f76a5445.png">

Do you happen to know how can I do it? or maybe somewhere I can read how to do it?

**Comment by jonathanurias96** [comment]
Hello @TahimiLeonBravo, I already added a PR that fixes this issue. Please review it and if theres anything that needs to be changed please let me know. Also feedback will be appreciated.
